plugins/modules/aci_aaa_user_role.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) <year>, <Name> (@<github id>)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_aaa_user_role
+short_description: Manage AAA user role (aaaUserRole) objects.
+description:
+- Manage AAA User Role configuration on Cisco ACI fabrics.
+options:
+  aaa_user:
+    description:
+    - Name of an existing AAA user
+    type: str
+    required: yes
+  domain_name:
+    description:
+    - Name of the user domain
+    type: str
+    required: yes
+  role_name:
+    description:
+    - Name of the AAA role
+    type: str
+  priv_type:
```

Comment:
can you check default behaviour of priv_type? 

looks like default value on APIC is readPriv, so when nothing provided it should default to this. if this is the case please add as part of description in this documentation part like below:

- The APIC defaults to C(read) when unset during creation.

Reply:
priv_type is required when present so there is no case where it is not set.